deployment/ansible/roles/nyc-trees.tasseo/templates/nyc-trees.js.j2

```diff
@@ -0,0 +1,50 @@
+var period = 10;
+var metrics = [
+    {
+        "alias": "django.percent-error",
+        "target": "scale(divideSeries(transformNull(sumSeries(statsite.counts.django.response.{4,5}*),0),transformNull(sumSeries(statsite.counts.django.response.{2,3}*),0)),100)",
```

Comment:
Fixed in https://github.com/azavea/nyc-trees/commit/d1554da050a45569db959be4fca8a353ca102e1f.